tagging_tools/gen_tag_info

```diff
@@ -41,6 +48,8 @@ def main():
     parser.add_argument("-e", "--entities", nargs='+', default=[],
                         required=False,
                         help="Entities file for policy")
+    parser.add_argument("--rv64", action="store_true", required=False,
```

Comment:
As discussed with @Hackmodford, the better generic argument name is `--arch`, which has options for `rv32` and `rv64`